platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheEncryptionIntegrationTest.groovy

```diff
@@ -52,11 +53,14 @@ class ConfigurationCacheEncryptionIntegrationTest extends AbstractConfigurationC
         encryptionKeyAsBase64 = Base64.encoder.encodeToString(encryptionKeyText.getBytes(StandardCharsets.UTF_8))
     }
 
-    def "configuration cache can be loaded without errors from #source using #encryptionTransformation"() {
+    def "configuration cache can be loaded without errors from #source using #encryptionTransformation and #keystoreType"() {
```

Comment:
❔ Does the unrolled test name read well with the wildcards?

Reply:
Well enough for my taste at least:

<img width="1522" height="486" alt="image" src="https://github.com/user-attachments/assets/2526dfac-281d-4cbc-a5ca-958b21a70c7f" />

Reply:
It's okay in the IDE, I guess, when you are looking at the test code immediately. It's less obvious when reading this on CI or Develocity. It would be a nice touch to make the wildcard more human-readable in the unrolled name.